README.md

```diff
@@ -23,9 +23,11 @@ The bot does the following:
 
 ## Setup
 
-1. Go to the [Google Cloud Console](https://console.cloud.google.com) and open the Cloud Shell.
+1. Go to the [Google Cloud Console](https://console.cloud.google.com) and create a new project
```

Comment:
```suggestion
1. Go to the [Google Cloud Console](https://console.cloud.google.com), create a new project and open this project.
```

When you open a Google Cloud project, Cloud Shell will automatically set the project ID for `gcloud`.